src/components/content/heroes/support.tsx

```diff
@@ -10,9 +10,15 @@ import { rgba } from 'polished';
  * to spread the content over the full available width.
  * That way we don't need any position absolute & friends.
  * */
-export const HeroContainer = styled.div`
+export const HeroContainer = styled.div<{ isBlog?: boolean }>`
   display: grid;
-  height: 520px;
+
+  ${({ isBlog }) =>
```

Comment:
Imo this change could be used for all page-types. But tbh., this is more a question in the direction of @georgiee